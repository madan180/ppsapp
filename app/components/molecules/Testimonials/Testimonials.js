/** * * Testimonials * */

// @flow
import { PureComponent } from 'react';
import styles from './Testimonials.style';
import withStyles from '../../../lib/withStyles';
import type { Props } from './types';

class Testimonials extends PureComponent<Props> {
  static defaultProps = {};

  render() {
    const { className } = this.props;

    return (
      <section className={`wrapper ${className}`}>
        <div className="inner">
          <header className="special">
            <h2>Faucibus consequat lorem</h2>
            <p>
              In arcu accumsan arcu adipiscing accumsan orci ac. Felis id enim aliquet. Accumsan ac
              integer lobortis commodo ornare aliquet accumsan erat tempus amet porttitor.
            </p>
          </header>
          <div className="testimonials">
            <section>
              <div className="content">
                <blockquote>
                  <p>
                    Nunc lacinia ante nunc ac lobortis ipsum. Interdum adipiscing gravida odio
                    porttitor sem non mi integer non faucibus.
                  </p>
                </blockquote>
                <div className="author">
                  <div className="image">
                    <img src="/static/images/pic01.jpg" alt="" />
                  </div>
                  <p className="credit">
                    - <strong>Jane Doe</strong> <span>CEO - ABC Inc.</span>
                  </p>
                </div>
              </div>
            </section>
            <section>
              <div className="content">
                <blockquote>
                  <p>
                    Nunc lacinia ante nunc ac lobortis ipsum. Interdum adipiscing gravida odio
                    porttitor sem non mi integer non faucibus.
                  </p>
                </blockquote>
                <div className="author">
                  <div className="image">
                    <img src="/static/images/pic03.jpg" alt="" />
                  </div>
                  <p className="credit">
                    - <strong>John Doe</strong> <span>CEO - ABC Inc.</span>
                  </p>
                </div>
              </div>
            </section>
            <section>
              <div className="content">
                <blockquote>
                  <p>
                    Nunc lacinia ante nunc ac lobortis ipsum. Interdum adipiscing gravida odio
                    porttitor sem non mi integer non faucibus.
                  </p>
                </blockquote>
                <div className="author">
                  <div className="image">
                    <img src="/static/images/pic02.jpg" alt="" />
                  </div>
                  <p className="credit">
                    - <strong>Janet Smith</strong> <span>CEO - ABC Inc.</span>
                  </p>
                </div>
              </div>
            </section>
          </div>
        </div>
      </section>
    );
  }
}
export default withStyles(Testimonials, styles);
export { Testimonials };
