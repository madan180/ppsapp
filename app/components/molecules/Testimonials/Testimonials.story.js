import React from 'react';
import { storiesOf } from '@storybook/react';
// import { action } from '@storybook/addon-actions';
import Testimonials from './index';

storiesOf('Testimonials', module).add('', () => <Testimonials>Hello Testimonials</Testimonials>);
