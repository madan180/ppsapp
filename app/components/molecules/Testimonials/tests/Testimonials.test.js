import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Testimonials from '../index';

describe('<Testimonials />', () => {
  let TestimonialsComponent = '';
  beforeEach(() => {
    TestimonialsComponent = shallow(<Testimonials>Test</Testimonials>);
  });

  test('should render correctly', () => {
    expect(TestimonialsComponent).toMatchSnapshot();
  });
});
