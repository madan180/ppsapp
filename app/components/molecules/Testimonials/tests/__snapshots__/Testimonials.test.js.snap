// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Testimonials /> should render correctly 1`] = `
<Testimonials
  className=""
>
  Test
</Testimonials>
`;
